tests/unit/test_utils.py

```diff
@@ -148,6 +149,14 @@ def create_last_run(config, counter):
     assert config.last_run(), "Some run was stored as last run"
 
 
+def test_workdir_env_var(tmpdir):
+    os.environ['TMT_RUN_DIR'] = str(tmpdir)
```

Comment:
See pytest's `monkeypatch` fixture, https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html#monkeypatching-environment-variables - `monkeypatch.setenv("TMT_RUN_DIR", str(tmpdir))` would be less prone to any surprise when confronted with pytest's workflow, and it'd even take care of undoing the patch.

Reply:
thanks, resolving, used monkeypatch